docs/maintainers/maintainers-guide.md

```diff
@@ -0,0 +1,152 @@
+---
+id: maintainers-guide-to-open-sauced
+title: 'Maintainers Guide to OpenSauced'
+sidebar_label: Maintainers Guide to OpenSauced
```

Comment:
The `sidebar_label` in some files have single or double quotes (e.g., in the `insights.md` file) and some don't.
Which one should we keep for consistency?

Reply:
Remove them.